RENOVATE.md

```diff
@@ -154,7 +155,8 @@ alerts, enabled by a previous preset, will use a `rangeStrategy` of `update-lock
 ```
 
 The first of these package rules will ensure that non-development dependency version constraints are widened when a
-newer version is available outside them. Widening the range of a development dependency makes little sense.
+newer version is available outside them. Widening the range of a development dependency makes little sense, so a bump
+strategy is used to ensure that development dependencies are kept at the most recent possible minor or patch versions.
  - `replace` Replace the range with a newer one if the new version falls outside it, and update nothing otherwise.
```

Comment:
This line would also need changing to reflect the use of bump instead of replace.